### PR TITLE
Add gift class

### PR DESCRIPTION
**Pull request by fla998**
#### What problem is this solving?

The free products are visible in the product cart and the user sees the dropdown and their price. It is necessary not to make visible this informations, to do this the "--gift" class has been added.

Cattura 1

Cattura 2

A "--gift" class was also created to change the position of "Omaggio"

Cattura 3

#### How to test it?

https://fminutillosprint18--itwhirlpool.myvtex.com/

### CODE REVIEW

**Comment by fla998** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by fla998** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Review by igorbrasileiro — APPROVED** [review]

**Comment by miguel-carrera** [comment]
@alvim @igorbrasileiro @darlenedms do we need more approvals to be able to merge this PR?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Add gift class in `QuantitySelector` and `Price`
```
Comment (igorbrasileiro) [inline CHANGELOG.md]:
```suggestion
### Added
- gift class in `QuantitySelector` and `Price`
```

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**